src/com/amplitude/api/DefaultNetworkClient.java

```diff
@@ -0,0 +1,33 @@
+package com.amplitude.api;
+
+import okhttp3.FormBody;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+import java.io.IOException;
+
+public class DefaultNetworkClient implements NetworkClient {
+
+    @Override
+    public Response uploadEvents(EventUploadRequest eventUploadRequest, OkHttpClient client) throws IOException, IllegalArgumentException {
+
+        String apiVersionString = "" + eventUploadRequest.getApiVersion();
+        String timestampString = "" + eventUploadRequest.getUploadTime();
+
+        FormBody body = new FormBody.Builder()
+                .add("v", apiVersionString)
```

Comment:
I left the same feedback in my review of the iOS version of this PR, but basically it feels weird of this logic to live inside the network client itself since this is just Amplitude's event server API. Regardless of the network client / protocol, the data format should be the same right? It almost feels like the FormBody should be created by the parent and passed into the client already formed, and the client just handles sending it to the proper location / method.

You can ignore all of this feedback if your custom network client has a completely different data format for your own endpoint